yarn.lock

```diff
@@ -6807,22 +6950,23 @@ fsevents@~2.1.2:
   linkType: hard
 
 "node-gyp@npm:latest":
-  version: 7.1.2
-  resolution: "node-gyp@npm:7.1.2"
+  version: 9.4.0
```

Comment:
look here

Reply:
Needle in a haystack!

Reply:
There's no way to keep the diff smaller?

Reply:
I don't think so :(

I think this should be safe enough though